From: "Barak A. Pearlmutter" <bap@debian.org>
Date: Thu, 3 Jan 2019 20:48:03 +0100
Subject: typos

---
 start.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/start.c b/start.c
index 8747361..53097ba 100644
--- a/start.c
+++ b/start.c
@@ -142,7 +142,7 @@ static char* optionsHelpArray[] = {
 "                                        instead of 6:37)",
 "    -F, --daf-yomi                     Output the Daf Yomi for the entire",
 "                                        date range",
-"    -g, --iso-8601                     Ouput ISO 8601 dates -- YYYY-MM-DD",
+"    -g, --iso-8601                     Output ISO 8601 dates -- YYYY-MM-DD",
 "                                            this overrides -y", 
 "    -h, --no-holidays                  Suppress default holidays",
 "    -i, --israeli                      Israeli holiday and sedra schedule",
